github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-presubmits.yaml

```diff
@@ -53,7 +53,7 @@ presubmits:
             - "/usr/local/bin/runner.sh"
             - "/bin/sh"
             - "-c"
-            - "make test-functional"
+            - "make build-builder cluster-up cluster-push-image cluster-sync test-functional CLUSTER_PREFIX='-p pull-kvp-functional-test'"
```

Comment:
So the prevailing consensus is that it makes sense to create a script file, and call that script file from here. This way you can independently modify the script and retest changes without having to modify the project-infra code. So I suggest you make a script file that basically does what this line does and call that script from here.

Reply:
I can also have those as prerequisites for `test-functional` target but it makes local testing annoying. And because I do need the change to unit tests anyway (`priviledged: true`), I thought I change this as well and have better developer experience.

Reply:
Yep, agree with @awels, you can just call the script from here and have whatever logic you want inside, this way the CI logic is kept in the same repo as the source code. In this case you can start with the whole `make` call, but later on you can update the CI logic from the target repo itself.